src/content/docs/tutorial-infrastructure-monitoring/investigate-spike-CPU.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Investigate a spike in load
+metaDescription: Solve system outages by investigating resource bottlenecks across your system infrastructure.
+---
+
+You can use the infrastructure hosts pages to troubleshoot different incidents within your system. In the last doc, we discussed the benefits of fully instrumenting your system with recommendations from the guided install, then walked through a general example of how and why you might filter down to a set of hosts. This doc builds off that skill and walks you through how you might troubleshoot a real-world example.  
+
+Let's say your customer support has notified you that a certain part of your site is down, namely the ability to complete purchase transactions. You may not have all the details, but two possibilities come to mind:
+
+* A new service deployment could be adding load to the servers or databases
+* A background job process could be creating the spike 
+
+Because you've instrumented your system, you have data reporting. But how do you find that data?
+
+## Objectives [#objectives]
+
+In this tutorial, you'll learn how to:
+
+* Find the cause of an unexpected load spike
+* Determine if resources are deprived because of a new deployment, or because of a job process
+* Locate the deployment or process in your stack
+
+## Investigate a sudden spike in load [#spike-load]
+
+ELEPHANT
+
+<Steps>
+    <Step>
+    ### Start with a filter set
+        
```

Comment:
could theoretically add context about using the 'Saved views' feature to save helpful filters of your hosts once you make one https://docs.newrelic.com/docs/new-relic-solutions/new-relic-one/core-concepts/search-filter-entities/#save